### ISSUE TITLE
Support for percentage width

### ISSUE THREAD

**Issue by lcsrinaldi** [danilowoz/react-content-loader#87]
I wanted a `svg` element that expanded to take all my `div` and only could achieve this passing `"100%"` to the `width` property. The problem is that the `viewBox` prop doesn't accept percentages and I received a lot of warnings on the console.

**Comment by danilowoz** [comment]
Hey @lcsrinaldi, thanks for opened this question, maybe someone has the same doubt.

But the attribute accepts only number ([see on the MDN](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/viewBox)), because the SVG needs a specific size.

What you should do is create a css class, like:
```css
.my-loader {
   width: 100%;
}
```

Sound good? Let me know if it solves your problem.

**Comment by lcsrinaldi** [comment]
Thanks for your answer @danilowoz, I've tried this and the problem is that even though the SVG element fills the whole `div` box, its elements don't.  

Then I tried two things, 1) achieve what I want with `preserveAspectRatio` which didn't work because the elements were all stretched and 2) to remove the `viewBox` property completely, which worked fine.

So maybe an option to not use the `viewBox`?
